x-pack/legacy/plugins/lens/public/types.ts

```diff
@@ -222,7 +217,7 @@ export interface VisualizationProps<T = unknown> {
 
 export interface SuggestionRequest<T = unknown> {
   // It is up to the Visualization to rank these tables
-  tables: TableSuggestion[];
+  table: TableSuggestion;
```

Comment:
So I think this is the summary of your change: the editor frame now makes a suggestion request per table, because the suggestion ranking does not need to do pairwise comparisons?

If so, please update the comment here.

Reply:
You are right, done.